terasoluna-gfw-functionaltest-web/src/main/resources/META-INF/spring/spring-mvc-exceptionhandling-change-default-status-code.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:context="http://www.springframework.org/schema/context"
+  xmlns:mvc="http://www.springframework.org/schema/mvc" xmlns:util="http://www.springframework.org/schema/util"
+  xmlns:aop="http://www.springframework.org/schema/aop"
+  xsi:schemaLocation="http://www.springframework.org/schema/mvc http://www.springframework.org/schema/mvc/spring-mvc.xsd
+        http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+        http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd
+        http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd
+        http://www.springframework.org/schema/aop http://www.springframework.org/schema/aop/spring-aop.xsd">
+
+  <!-- Common Definition -->
+  <import resource="classpath:/META-INF/spring/spring-mvc-common.xml" />

```

Comment:
- [ ] With `spring-mvc-common.xml` setting, there is an unexpected request mapping because one controller is component scanned with multiple servlets. In those mappings, there is a possibility of doing unexpected behavior.
I think it's better to disassemble the `ExceptionHandlingController` and set it to be only the request mapping required for the servlet.
I will create another issue.